src/debug.c

```diff
@@ -1099,6 +1112,11 @@ void logStackContent(void **sp) {
 /* Log dump of processor registers */
 void logRegisters(ucontext_t *uc) {
     serverLog(LL_WARNING|LL_RAW, "\n------ REGISTERS ------\n");
+#define NOT_SUPPORTED() do {\
+    UNUSED(uc);\
+    serverLog(LL_WARNING,\
+              "  Dumping of registers not supported for this OS/arch");\
+} while(0)
```

Comment:
same here, i'd be ok with just adding `UNUSED(uc)`.
the the very fact the register dump is missing seems enough (the log print is unnecessary).

Reply:
Was trying to keep the previous behavior. If we scrap the logging then we can just add `UNUSED(uc);` since the there's no return value in this function.